### DISCUSSION TITLE
🖋: トラブルシューティングの存在に気づきにくい

### DISCUSSION THREAD

**Discussion by takei0w0** [ws-4020/mobile-app-crib-notes#147] [Ideas]
学習コンテンツFBより

- トラブルシューティングの存在に気づきにくい
- 学習用コンテンツのメニューに有ると嬉しい
- 環境構築絡みのトラブルシューティングはGEttingStartedのところにほしい

トラブルシューティングの名前を変える #115 とも関連してどう誘導するかは検討したほうがよいかも

**Comment by creature-water-valley** [comment]
#1266 のように、該当のトラブルが発生しやすい箇所に、リンクを追加する方法としました。
